### DISCUSSION TITLE
NEXTSTEP Clean Code with JavaScript 2기 회고 및 후기 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#14] [Announcements]
# NEXTSTEP Clean Code with JavaScript 2기 회고 및 후기 | 정현수 기술 블로그

2022-03-08 - 2022-04-26 학습을 진행하면서 느낀 것들

https://junghyeonsu-dev.vercel.app/posts/NextStep-clean-code-with-javascript-2th

**Comment by pocojang** [comment]
오우 현수님
솔직한 후기 감사합니다!
정말 취준생으로서 큰 돈을 쓰시고 적극적으로 참여해주셔서 정말 감사해요 :)

**Reply by junghyeonsu** [reply]
포코님 감사합니다!!
나중에 프론트엔드 교육 프로그램이 하나 더 나오면 듣고 싶네요..
존버하고 있겠습니당..!
